packages/react-components/src/components/VideoTile.tsx

```diff
@@ -96,9 +95,9 @@ export const VideoTile = (props: VideoTileProps): JSX.Element => {
   } = props;
 
   const placeHolderProps = { userId, displayName, noVideoAvailableAriaLabel };
-  const theme = useTheme();
+
   return (
-    <Stack className={mergeStyles(rootStyles, { background: theme.palette.neutralLighter }, styles?.root)}>
```

Comment:
The theme might have been used to reconcile the palette for dark theme. Does this look ok in dark theme and just this is no longer needed? (similar with the below reference to palette).